contracts/FungibleTokenSwitchboard.cdc

```diff
@@ -126,18 +128,56 @@ pub contract FungibleTokenSwitchboard {
         /// capability, rather than the `Type` from the reference borrowed from
         /// said capability
         /// 
-        pub fun addNewVaultWrapper(capability: Capability<&{FungibleToken.Receiver}>, type: Type) {
+        pub fun addNewVaultWrapper(capability: Capability<&{FungibleToken.Receiver}>, 
+                                                                        type: Type) {
             // Check if the capability is working
             assert(capability.check(), message: "The passed capability is not valid")
             // Use the type parameter as key for the capability
             self.receiverCapabilities[type] = capability
             // emit the event that indicates that a new capability has been 
             // added
             emit VaultCapabilityAdded(type: type,
-                                    switchboardOwner: self.owner?.address, 
-                                    capabilityOwner: capability.address)
+                                               switchboardOwner: self.owner?.address, 
+                                                 capabilityOwner: capability.address)
+        }
+
+        /// Adds zero or more new fungible token receiver capabilities to the  
+        /// switchboard resource specifying which `Type`s of `@FungibleToken.Vault`s  
+        /// can be deposited to it. Use it to include in your switchboard "wrapper"
+        /// receivers such as a `@TokenForwarding.Forwarder`. It can also be
+        /// used to overwrite the types attached to certain capabilities without 
+        /// having to remove those capabilities first.
+        ///                    
+        /// @param paths: The paths where the public capabilities are stored.
+        /// @param types: The types of the fungible token to be deposited on each path.
+        /// @param address: The address of the owner of the capabilities.
+        /// 
+        pub fun addNewVaultsWrappersByPath(paths: [PublicPath], types: [Type], 
+                                                                  address: Address) {
```

Comment:
maybe `pub fun addNewVaultWrappersByPath` ?

Reply:
also why don't we send here [Type] and [Capability<&FungibleTokenReceiver>]

Reply:
yep that sounds better thx! 

Honestly I just thought it will be nice to have the same methods for capabilities stored under their own type and for wrappers, so I mixed `addNewVaultsByPath` and `addNewVaultWrapper`